frontend/src/components/H2.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+
+class H2 extends Component {
+  render() {
+    return (
+      <div className="sub-headline">
+        <h2>{this.props.h2}</h2>
```

Comment:
let's use `this.props.children` here instead

Reply:
Implemented it for all relevant components (H1, H2, Button)